assets/js/components/Dashboard/PlaceRowAccordion.tsx

```diff
@@ -23,11 +23,12 @@ type ScreenGroup = {
 
 const groupScreens = (screens: Screen[]): ScreenGroup[] => {
   const inlineScreens = screens.filter((screen) =>
-    ["busway_v2", "solari"].includes(screen.type),
+    ["busway_v2", "solari", "elevator_v2"].includes(screen.type),
```

Comment:
Going the flexbox route should be easy. Just need to add `flex-wrap`. Wondering if it would be helpful to group screens of the same type into each inlined group. Without it, two sections and two elevator screens leads to 3 screens in one row and one in another. Grouping first would split them evenly. What do you think?

Reply:
I think screens of the same type should be grouped, yes. It feels like an oversight that they aren't (maybe we only had one "inline" screen type at first).

Reply:
Yeah only Solari/Sectionals were inline before this. Fixed them up so they are grouped by type.